src/m_menu.c

```diff
@@ -3343,7 +3347,7 @@ setup_menu_t stat_settings2[] =
   {"CROSSHAIR",S_SKIP|S_TITLE,m_null,M_X,M_Y+stat2_title3*M_SPC },
 
   {"ENABLE CROSSHAIR",      S_CHOICE,m_null,M_X,M_Y+stat2_xhair*M_SPC,       {"hud_crosshair"}, 0, M_UpdateCrosshairItems, crosshair_str},
-  {"COLOR BY PLAYER HEALTH",S_YESNO, m_null,M_X,M_Y+stat2_xhairhealth*M_SPC, {"hud_crosshair_health"}},
+  {"COLOR BY HEALTH",       S_CHOICE,m_null,M_X,M_Y+stat2_xhairhealth*M_SPC, {"hud_crosshair_health"}, 0, NULL, crosshair_health_str},
```

Comment:
While crosshair coloring by target health itself is disabled on Strict Mode, the setup menu item is not disabled since it uses the same variable. which might be used for coloring by player health which apparently _is_ allowed in Strict Mode. Should I add a separate variable, which of course means a separate setup menu item?